walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -131,32 +127,32 @@ Verify that you followed each step in the procedure above.  If you are still hav
 ****
 
 [time=15]
-== Protect the GraphQL showcase app using the Red Hat SSO
+== Protect the showcase app using the Red Hat SSO
 
-In this task, you use the Red Hat SSO to protect both the client (PWA) and the data sync server.
+In this task, we will protect both the frontend and the backend of the {data-sync-showcase-app} with the {customer-sso-name}.
```

Comment:
I'm not big fan of the `protect` world as it is not universally translating to `add authz`. Not big issue